Casks/android-sdk.rb

```diff
@@ -1,9 +1,9 @@
 cask 'android-sdk' do
-  version '4333796'
-  sha256 'ecb29358bc0f13d7c2fa0f9290135a5b608e38434aad9bf7067d0252c160853e'
+  version '6200805'
+  sha256 '23f0626336a98d70aff7311a73292026af31bc577c6f06b509cd4ad33752313e'
 
   # dl.google.com/android/repository was verified as official when first introduced to the cask
-  url "https://dl.google.com/android/repository/sdk-tools-darwin-#{version}.zip"
+  url "https://dl.google.com/android/repository/commandlinetools-mac-#{version}_latest.zip"
   name 'android-sdk'
   homepage 'https://developer.android.com/index.html'
```

Comment:
It is a really cosmetic suggestion, but now it is a good time to do it:
```suggestion
  homepage 'https://developer.android.com/'
```